tests/legacy-cli/e2e/tests/build/polyfills.ts

```diff
@@ -1,18 +1,26 @@
+import { oneLineTrim } from 'common-tags';
 import { expectFileToMatch } from '../../utils/fs';
 import { ng } from '../../utils/process';
-import { oneLineTrim } from 'common-tags';
+import { expectToFail } from '../../utils/utils';
 
-export default function () {
-  // TODO(architect): Delete this test. It is now in devkit/build-angular.
+export default async function () {
+    await ng('build');
+    // files were created successfully
+    await expectFileToMatch('dist/test-project/polyfills.js', 'core-js/es7/reflect');
+    await expectFileToMatch('dist/test-project/polyfills.js', 'zone.js');
+    expectFileToMatch('dist/test-project/index.html', oneLineTrim`
+      <script type="text/javascript" src="runtime.js"></script>
+      <script type="text/javascript" src="ie-support.js" nomodule></script>
+      <script type="text/javascript" src="polyfills.js"></script>
+    `);
 
-  return Promise.resolve()
-    .then(() => ng('build'))
+    await ng('build', '--aot');
     // files were created successfully
-    .then(() => expectFileToMatch('dist/test-project/polyfills.js', 'core-js'))
-    .then(() => expectFileToMatch('dist/test-project/polyfills.js', 'zone.js'))
-    // index.html lists the right bundles
-    .then(() => expectFileToMatch('dist/test-project/index.html', oneLineTrim`
+    await expectToFail(() => expectFileToMatch('dist/test-project/polyfills.js', 'core-js/es7/reflect'));
```

Comment:
@clydin can you please make this assertion more explicit? we don't want to pass through this if expectFileToMatch fails for other reasons (like file not found).

Reply:
Updated.
The polyfills file now needs to also exist and the AOT file size needs to be less than the JIT file size.